package.json

```diff
@@ -6,9 +6,11 @@
   "version": "1.3.2",
   "dependencies": {
     "commander": "^2.6.0",
+    "debug": "^2.1.1",
     "denodeify": "^1.2.0",
     "es6-promise": "^2.0.1",
     "haikro": "^1.9.0",
+    "heroku-client": "^1.9.1",
```

Comment:
when [this build finishes](https://travis-ci.org/matthew-andrews/haikro/builds/50318687) you should be able to do:-

```js
var provision = require('haikro/lib/create');
create({ token: token, app: 'my-app-name', region: 'eu', organization: 'financial-times' })
  .then(function() {
    // success
  });
```